android/src/test/java/com/magicleap/magicscript/scene/nodes/layouts/PageViewNodeTest.kt

```diff
@@ -96,6 +97,8 @@ class PageViewNodeTest {
     }
 
     private fun createNode(props: JavaOnlyMap): PageViewNode {
-        return PageViewNode(props, layoutManager)
+        return PageViewNode(props, layoutManager).also {
+            it.addContent(NodeBuilder().build())
```

Comment:
Probably we should not add children before node is built.

Reply:
work fine, the test is passing and the child is added

Reply:
Yes, but in particular in tests we should "show" the proper use case. This is natural flow of magic-scrip and all tests have followed this pattern so far.